ftw/file/utils.py

```diff
@@ -25,3 +26,70 @@ def redirect_to_download_by_default(context):
             request.other['disable_border'] = border_was_force_disabled
         if border_was_force_enabled:
             request.other['enable_border'] = border_was_force_enabled
+
+
+class FileMetadata(object):
+    """Handles some metadata of an object
```

Comment:
can you better explain why this class exists?
this comment is not really helpful